src/Raven.Client/ServerWide/DatabaseTopology.cs

```diff
@@ -213,9 +214,23 @@ public bool RelevantFor(string nodeTag)
             if (Promotables.Contains(myTag)) // if we are a promotable we can't have any destinations
                 return destinations;
 
-            var nodes = Members.Concat(Rehabs);
+            if (Rehabs.Contains(myTag)) // if I'm rehab my only destination is my mentor
```

Comment:
that is the other way around form what I think should happen, no?
If I'm on Rehab, I still want to connect _out_ to whoever I can.
However, this is from multiple members connecting to rehab, no?

Reply:
If `Rehabs` contains my tag... I will connect my mentor.

Reply:
I know, my question is why should the rehab node connect to everyone?
Why wait until it is out of rehab?

Reply:
Rehab shouldn't connect to every one, only to his mentor.
Since, it might happened that the Rehab node contain some data that the others doesn't have.